dhtnew.cpp

```diff
@@ -63,14 +64,20 @@ DHTNEW::DHTNEW(uint8_t pin)
   #endif
 };
 
+uint8_t DHTNEW::getType()
+{
+  if (_type == 0) read();
```

Comment:
Should it not call **_read()**
think this will be an exploding stack if _type == 0